src/Im2Col.h

```diff
@@ -70,8 +69,7 @@ template <>
 void im2col<1>(const int channels,
                const std::vector<net_t>& input,
                std::vector<float>& output) {
-    constexpr unsigned int boardsize = 19;
-    auto outSize = size_t{channels * boardsize * boardsize};
+    auto outSize = size_t{channels * static_cast<size_t>(BOARD_SQUARES)};
```

Comment:
Does this actually need both size_t casts?

Reply:
If I don't cast the value, it raises a warning which makes it fail the AppVeyor check.